README.adoc

```diff
@@ -83,4 +83,4 @@ added after the original pull request but before a merge.
   other target branch in the main project).
 * When writing a commit message please follow http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html[these conventions],
   if you are fixing an existing issue please add `Fixes gh-XXXX` at the end of the commit
-  message (where XXXX is the issue number).
\ No newline at end of file
+  message (where XXXX is the issue number).
```

Comment:
This added newline character at EOF, and was done by GitHub online editor automatically.